packages/core/src/account/smartContractAccount.ts

```diff
@@ -214,7 +215,7 @@ export async function toSmartContractAccount<
 export async function toSmartContractAccount({
   transport,
   chain,
-  entryPoint,
+  entryPoint = getEntryPoint(chain),
```

Comment:
we should leave this as required for this function as it was before because this way it can be easy for someone using `toSmartContractAccount` to omit it.

imagine for LA we forget to pass the EP down through to the toSCA call. user passes in EP with address override but still is valid 0.6. there will be no type errors in the LA definition or in the caller's call, but their override doesn't get used

Reply:
make sense